src/components/TopAppBar/TopAppBar.style.ts

```diff
@@ -0,0 +1,47 @@
+import { css, SerializedStyles } from '@emotion/core';
+import { Theme } from '../../theme';
+import { BgColorType } from './TopAppBar.types';
+import { flexCenter } from '../../theme/functions';
+import { pickTextColorFromSwatches } from '../../theme/palette';
+
+const DEFAULT_NAVBAR_HEIGHT = 62;
+
+interface StyleProps {
+  bgColor: BgColorType;
+}
+
+const topAppBarWrapper = ({ bgColor }: StyleProps) => (theme: Theme): SerializedStyles => css`
+  ${flexCenter};
+  position: relative;
+  justify-content: space-between;
+  background-color: ${theme.utils.getColor(bgColor.type, bgColor.variant)};
+  color: ${pickTextColorFromSwatches(bgColor.type, bgColor.variant)};
+  height: ${DEFAULT_NAVBAR_HEIGHT}px;
+  padding: ${theme.spacing.sm} ${theme.spacing.md};
```

Comment:
this padding must change to `padding: 0 ${theme.spacing.md};` as the components inside has the `sm`. If you put this the height of the topbar exceeds the 62px which is the design